Example/AppDelegate+CarPlay.swift

```diff
@@ -153,6 +154,44 @@ extension AppDelegate: CarPlayManagerDelegate {
 
 @available(iOS 12.0, *)
 extension AppDelegate: CarPlaySearchControllerDelegate {
+    struct RecentsHolder {
+        static var _recentItems: [RecentItem] = RecentItem.loadDefaults()
+        static var _recentSearchItems: [CPListItem]? = []
+        static var _recentSearchText: String? = ""
+    }
+    
+    var recentItems: [RecentItem] {
+        get {
+            return RecentsHolder._recentItems
+        }
+        set(newValue) {
+            RecentsHolder._recentItems = newValue
+        }
+    }
```

Comment:
Do we expect these computed properties to get any more complex in the future, or can we turn them into ordinary stored properties, eliminating the need for RecentsHolder and its properties?

Reply:
I didn’t notice that these properties are declared in an extension. It would be fine to declare three ordinary stored properties as part of the main AppDelegate declaration and use them within this extension. We’re already quite from from the point where this file would be self-contained anyways.

Reply:
Moving recentSearchItems would require AppDelegate to be available for iOS 12 or later since it is of type `[CPListItem]`.

Reply:
As a workaround, the property could be typed as `[AnyObject]` and cast to `[CPListItem]` where necessary.